### PR TITLE
Added Sigasi/Zeus/Ceneka FPGA event

### PR DESCRIPTION
**Pull request by tivervac**

### CODE REVIEW

File: content/events/16-17/spelenmetfpgas.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Spelen met FPGAs
+description: Sigasi geeft een introductie tot FPGAs
+time: 27-04-2017
+created_at: 12-04-2017
+---
+FPGAs zijn hipper dan ooit. Alle grote spelers gebruiken deze flexibele chips om een specifieke voorsprong te krijgen op hun concurrenten. Toch zijn FPGAs nog steeds vrij exotisch en weinig toegankelijk.
```
Comment (werthen) [inline content/events/16-17/spelenmetfpgas.md]: Maybe quickly introduce/explain what FPGA's are specifically? This explanation is very very broad

File: content/events/16-17/spelenmetfpgas.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (werthen) [inline content/events/16-17/spelenmetfpgas.md]: Headers as follows are also needed (note the exact timing):
```
time: 23-02-2017 18:30
end: 23-02-2017 23:00
location: A3, S9
locationlink: Sterre,Ghent Belgium
```

File: content/events/16-17/spelenmetfpgas.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+title: Spelen met FPGAs
+description: Sigasi geeft een introductie tot FPGAs
+time: 27-04-2017
```
Comment (mivdnber) [inline content/events/16-17/spelenmetfpgas.md]: This shows as midnight on the event page:

screenshot 2017-04-13 15 51 35


I guess this needs hours and minutes. A location wouldn't hurt either.

Comment (tivervac) [inline content/events/16-17/spelenmetfpgas.md]: Once they have been decided, they will be added ;)

**Pull request merged**